### ISSUE TITLE
VS 17.8.2 compiler not being recognized

### ISSUE THREAD

**Issue by lucz-volue** [github/codeql#14978]
After updating visual studio to 17.8.2  creating database fails, looks like new VS compiler is not recognized:

```
Command: E:\codeql-win64\codeql\cpp\tools\win64\extractor.exe --mimic "c:\program files\microsoft visual studio\2022\professional\vc\tools\msvc\14.38.33130\bin\hostx64\x64\cl.exe" @C:\Users\[UserName]\AppData\Local\Temp\MSBuildTemp[UserName]\tmp599df9d71d68459c8d051f2f1a791ace.rsp
Warning: c:\program files\microsoft visual studio\2022\professional\vc\tools\msvc\14.38.33130\bin\hostx64\x64\cl.exe does not look like a compiler. If it is infact a compiler, please contact GitHub.
```

**Comment by jketema** [comment]
Hi @lucz-volue,

We are aware. I'm going to close this as a duplicate of https://github.com/github/codeql/issues/14927.

**Comment by jketema** [comment]
We believe we have fixed this. The fix should be part of CodeQL 2.15.4, which should be released a few days from now.
